components/desktop/mate/mate-session-manager/files/metacity.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
+Type=Application
+Exec=/usr/bin/metacity
```

Comment:
It's strange that you have to start metacity in such way. As  I understand, caja window manager should be used by default. What's going wrong with it?